aicsimageio/readers/reader.py

```diff
@@ -20,6 +20,31 @@
 
 ###############################################################################
 
+# Global variable to inform reader to use _read_delayed or _read_immediate
+# Useful in sitations where you may be on a cluster with very few workers
+# but still using Dask for other operations.
+# I.E. GPU clusters where you don't care about the IO of the file so much as the
+# training and application of a model
+USE_DASK = True
+
+def enable_dask():
```

Comment:
how about one single api instead of two?  set_use_dask(boolean). or set_dask_enabled(boolean)

Reply:
Fixed